src/compiler/diagnosticMessages.json

```diff
@@ -3571,7 +3571,10 @@
         "category": "Error",
         "code": 7038
     },
-
+    "Mapped object type implicitly has an 'any' template type.": {

```

Comment:
Probably want to switch from `'any'` to `'{0}'`

Reply:
Isn't this error strictly for an implicit any type? Is there any other possible value if you leave out the type annotation in a mapped type?

Reply:
Changing it produces `Mapped object type implicitly has an 'undefined' template type` :grimacing:

Reply:
i think the previous behavior was correct. sorry about that. they type is guaranteed to be `any`. we pass in the type if it can be `any` | `any[]` for instance. but in this case that is not needed. i would keep the old message.
sorry for the wrong suggestion here.

Reply:
Ok, I reverted the change.